lblod/frontend-form-builder
# SFB-72: start of custom codelist

## ID
 [SFB-72](https://binnenland.atlassian.net/browse/SFB-72)

 ## Description

The user wants to have a new interface where they want to see and edit code lists. The generic code lists may not be edited but the others can. I created a new page for showing the code lists and viewing them.

This is behind a feature flag so no one will notice this when it is off.

The next step in another PR is to start creating new forms and editing them.

 ## Type of change


 ## How to test

1. Set the feature flag to false
2. Go to codelijsten in the ui
3. View all lists
4. See details of a list and go back

 ## Links to other PR's

 - /
